docs/develop/07_dApp/04_verifier.md

```diff
@@ -34,22 +37,26 @@ In this guide the website requests an email address that is owned by the DID.
 For that it uses the Email CType.
 You can search through existing CTypes in the [CType Index](https://github.com/KILTprotocol/ctype-index).
 
+<TsJsSnippet funcEnd="console">
   {EmailCtype}
+</TsJsSnippet>
 
 After settled on a CType, the server can build the request for the visitor.
 Since we want to ensure that the presentation of the credential is fresh, the server first has to create a random challenge.
 The presentation must include this challenge and since it's random, the presentation must be created and signed from scratch.
 This ensures that it's not possible to record a presentation and just send this, pretending to be the owner of the DID.
 The challenge can be generated using the polkadot crypto utilities:
 
-<TsJsBlock>
+<CodeBlock language="js">
```

Comment:
this was not even TS, so why have the switcheroo?

Reply:
Also I think it's more consistent if all code snippets show the Ts-JS switch.

Reply:
Mhh you're, doesn't hurt and afaik you set the language once for all examples, right?

Reply:
Correct.